sdk/program/src/lib.rs

```diff
@@ -493,7 +493,6 @@ pub mod incinerator;
 pub mod instruction;
 pub mod keccak;
 pub mod lamports;
-pub mod last_restart_slot;
```

Comment:
looks like last_restart_slot.rs still needs to be deleted

Reply:
Ah right, thanks for noticing!